django_project/certification/views/course_type.py

```diff
@@ -214,10 +214,7 @@ def get_queryset(self):
         """
 
         qs = CourseType.objects.all()
-        if self.request.user.is_staff:
-            return qs
-        else:
-            return qs.filter(creator=self.request.user)
+        return qs
```

Comment:
Shouldn't we be filtering by organization admins? And for views above? I guess it is ok for any user to view course types but they should not be able to edit unless they are staff or org admin?